packages/squidex/schema/crn/schemas/events.json

```diff
@@ -9,7 +9,7 @@
-      "update": "const teamsSpeakersSet = new Set();\n\nif (\n    ctx.data.speakers\n    && ctx.data.speakers.iv\n) {\n    for (const speaker of ctx.data.speakers.iv) {\n        if (!speaker.team && !speaker.user) {\n            return reject('You must select team or user or both for spekaer');\n        }\n        \n        if (\n            speaker.team.length === 1\n            && speaker.user.length === 1\n        ) {\n            const teamSpeaker = `${speaker.team[0]}-${speaker.user[0]}`;\n            if (teamsSpeakersSet.has(teamSpeaker)) {\n                return reject('Duplicates with same team and speaker are not allowed');\n            }\n            \n            teamsSpeakersSet.add(teamSpeaker);\n            \n            getReference(speaker.user[0], function (users) {\n              if (users && users.length === 1 && users[0].SchemaId.name === 'external-authors') {\n                  return reject('Selecting team for speaker is not allowed when user is external author');\n              }\n            });\n        }     \n    }\n}"
+      "update": "const teamsSpeakersSet = new Set();\n\nif (\n    ctx.data.speakers\n    && ctx.data.speakers.iv\n) {\n    for (const speaker of ctx.data.speakers.iv) {\n        if (!speaker.team && !speaker.user) {\n            return reject('You must select team or user or both for spekaer');\n        }\n        \n        if (\n            speaker.team.length === 1\n            && speaker.user.length === 1\n        ) {\n            const teamSpeaker = `${speaker.team[0]}-${speaker.user[0]}`;\n            if (teamsSpeakersSet.has(teamSpeaker)) {\n                return reject('Duplicates with same team and speaker are not allowed');\n            }\n            \n            teamsSpeakersSet.add(teamSpeaker);\n            \n            getReference(speaker.user[0], function (users) {\n              if (users && users.length === 1 && users[0].SchemaId.name === 'external-authors') {\n                  return reject('Selecting team for speaker is not allowed when user is external author');\n              }\n            });\n        }     \n    }\n}\n\nif (ctx.data.videoRecording.iv !== ctx.oldData.videoRecording.iv) {\n    ctx.data.videoRecordingUpdatedAt.iv = new Date().toISOString().split('.')[0]+\"Z\";\n    replace();\n}\n"
```

Comment:
have you tried using just
```javascript
ctx.data.videoRecordingUpdatedAt.iv = new Date().toISOString();
```
?

Reply:
nice! I haven't tried that before! it works perfectly :)